set_bucket_tags/app.py

```diff
@@ -55,31 +59,94 @@ def get_principal_id(tags):
 
 def get_synapse_user_profile(synapse_id):
   '''Get synapse user profile data'''
-  syn = synapseclient.Synapse()
+  syn = get_synapse_client()
   user_profile = syn.getUserProfile(synapse_id)
   log.debug(f'Synapse user profile: {user_profile}')
-
   return user_profile
 
 
-def get_synapse_tags(user_profile):
-  '''Derive synapse tags from synapse user profile data'''
-
+def get_ssm_parameter(name):
+  '''Get an parameter from the SSM parameter store'''
+  client = boto3.client('ssm')
+  parameter = client.get_parameter(Name=name)
+  log.debug(f'Synapse ssm parameter: {parameter}')
```

Comment:
well i know we not gonna get secrets in this lambda because there's no policy to allow access to KMS keys therefore no Secure strings.  however i can see your point about this and don't mind removing this log.